src/cascadia/TerminalConnection/init.cpp

```diff
@@ -31,4 +33,5 @@ BOOL WINAPI DllMain(HINSTANCE hInstDll, DWORD reason, LPVOID /*reserved*/)
     return TRUE;
 }
 
+#pragma warning(suppress:26485) // Array-to-pointer decay might technically be avoidable, but this is elegant and clean.

```

Comment:
we may want to do this in the header so all consumers benefit

Reply:
ok.